From 584e0b01b5be077cbf1e7d06d0eb2432c755121b Mon Sep 17 00:00:00 2001
From: Wang Long <wanglong@laoqinren.net>
Date: Sun, 21 May 2017 22:28:30 +0800
Subject: [PATCH 1/2] add tmux.conf

Signed-off-by: Wang Long <wanglong@laoqinren.net>
---
 .tmux.conf | 41 +++++++++++++++++++++++++++++++++++++++++
 1 file changed, 41 insertions(+)
 create mode 100644 .tmux.conf

diff --git a/.tmux.conf b/.tmux.conf
new file mode 100644
index 0000000..2c2d40d
--- /dev/null
+++ b/.tmux.conf
@@ -0,0 +1,41 @@
+# Tmux settings
+
+# bind a reload key
+bind r source-file ~/.tmux.conf ; display-message "Config reloaded.."
+
+# Set XTerm key bindings
+setw -g xterm-keys on
+
+# Set colors
+set-option -g default-terminal "screen-256color"
+
+# Count sessions start at 1
+set -g base-index 1
+
+# Use vim bindings in copy mode
+setw -g mode-keys vi
+
+# Remap window navigation to vim
+unbind-key j
+bind-key j select-pane -D
+unbind-key k
+bind-key k select-pane -U
+unbind-key h
+bind-key h select-pane -L
+unbind-key l
+bind-key l select-pane -R
+
+# Set the title bar
+set -g set-titles on
+set -g set-titles-string '#(whoami) :: #h :: #(curl ipecho.net/plain;echo)'
+
+# Set status bar
+set -g status-utf8 on
+set -g status-bg black
+set -g status-fg white
+set -g status-interval 5
+set -g status-left-length 90
+set -g status-right-length 60
+set -g status-left "#[fg=Green]#(whoami)#[fg=white]::#[fg=blue]#(hostname -s)#[fg=white]::#[fg=yellow]#(curl ipecho.net/plain;echo)"
+set -g status-justify centre
+set -g status-right '#[fg=green][#[fg=cyan]%Y-%m-%d#[fg=green]]'
-- 
2.7.4

